src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -274,8 +275,13 @@ export default {
 		}
 	},
 	render: function(createElement) {
+		// Add the root icon to the first breadcrumb
 		const first = this.$slots.default[0]
-		first.componentOptions.propsData.icon = this.rootIcon
+		if (first) {
```

Comment:
Does it make sense to render at all if there is no first?
Also be careful, the first item might not be the BreadCrumb (can be a space, some text, a comment...)